docs/admin/scale.md

```diff
@@ -123,70 +31,171 @@ Since Coder's performance is highly dependent on the templates and workflows you
+apps.
+
+### Parameters
+
+The _scaletest-runner_ offers the following configuration options:
+
+- Workspace template selecting Kubernetes cluster size:
+  minimal/small/medium/large (_default_: minimal)
+- Number of workspaces
+- Wait duration between scenarios or staggered approach
+
+The template exposes parameters to control the traffic dimensions for SSH
+connections, workspace apps, and dashboard tests:
+
+- Traffic duration of the load test scenario
+- Traffic percentage of targeted workspaces
+- Bytes per tick and tick interval
+- _For workspace apps_: modes (echo, read random data, or write and discard)
+
+Scale testing concurrency can be controlled with the following parameters:
+
+- Enable parallel scenarios - interleave different traffic patterns (SSH,
+  workspace apps, dashboard traffic, etc.)
+- Workspace creation concurrency level (_default_: 10)
+- Job concurrency level - generate workspace traffic using multiple jobs
+  (_default_: 0)
+- Cleanup concurrency level
+
+### Kubernetes cluster
+
+Depending on the traffic projections, operators can deploy different sample
+clusters to perform scale tests. It is recommended to learn how to operate the
+scaletest-runner before running it against the staging cluster (or production at
+your own risk).
+
+There are a few cluster options
+[available](https://github.com/coder/coder/tree/main/scaletest/templates/scaletest-runner):
+
+| Cluster size | vCPU | Memory | Persisted storage | Details                                               |
+| ------------ | ---- | ------ | ----------------- | ----------------------------------------------------- |
+| minimal      | 1    | 2 Gi   | None              |                                                       |
+| small        | 1    | 1 Gi   | None              |                                                       |
+| medium       | 2    | 2 Gi   | None              | Medium-sized cluster offers the greedy agent variant. |
+| large        | 4    | 4 Gi   | None              |                                                       |
+
+#### Greedy agent
+
+The greedy agent variant is a template modification that makes the Coder agent
+transmit large metadata (size: 4K) while reporting stats. The transmission of
+large chunks puts extra overhead on coderd instances and agents when handling
+and storing the data.
+
+Use this template variant to verify limits of the cluster performance.
+
+### Observability
+
+During scale tests, operators can monitor progress using a Grafana dashboard.
+Coder offers a comprehensive overview
+[dashboard](https://github.com/coder/coder/blob/main/scaletest/scaletest_dashboard.json)
+that can seamlessly integrate into the internal Grafana deployment.
+
+This dashboard provides insights into various aspects, including:
+
+- Utilization of resources within the Coder control plane (CPU, memory, pods)
+- Database performance metrics (CPU, memory, I/O, connections, queries)
```

Comment:
Note: the coder deployment needs to have `CODER_PROMETHEUS_COLLECT_DB_METRICS=true`

Reply:
Noted